plugins/woocommerce/client/blocks/tests/e2e/global-setup.ts

```diff
@@ -68,6 +68,7 @@ async function globalSetup() {
 
 	try {
 		await wpCLI( `db import ${ DB_EXPORT_FILE }` );
+		await wpCLI( `db repair` );
```

Comment:
We also import the DB [here](https://github.com/woocommerce/woocommerce/blob/15a2a18e56dd461be91ff7f3d64302018af09842/plugins/woocommerce/client/blocks/tests/e2e/utils/test.ts#L141), should we do the same?

Reply:
Good point! I added it with b922cfd.